config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :exercises
```

Comment:
Já que não está sendo usado todos os endpoints de `exercises` você pode excluir algumas das resources geradas automaticamente: https://guides.rubyonrails.org/routing.html#restricting-the-routes-created